python/desc/imsim/camera_info.py

```diff
@@ -73,3 +74,29 @@ def mosaic_section(amp_info):
         ymin = 0 if yseg == 1 else height
         return afw_geom.Box2I(afw_geom.Point2I(xmin, ymin),
                               afw_geom.Extent2I(width, height))
+
+
+def getHourAngle(observatory, mjd, ra):
+    """
+    Compute the local hour angle of an object for the specified
+    MJD and RA.
+
+    Parameters
+    ----------
+    observatory: lsst.sims.GalSimInterface.LsstObservatory
+        Extension of lsst.afw.coord.Observatory object.
+    mjd: float
+        Modified Julian Date of the observation.
+    ra: float
+        Right Ascension (in degrees) of the object.
+
+    Returns
+    -------
+    float: hour angle in degrees
+    """
+    time = astropy.time.Time(mjd, format='mjd',
+                             location=observatory.getLocation())
+    # Get the local apparent sidereal time.
+    last = time.sidereal_time('apparent').degree
+    ha = last - ra
```

Comment:
Are we allowing negative HA, or are we going with 'If result is negative, add 360 degrees. If result is greater than 360, subtract 360 degrees.'?

Reply:
Traditionally HA is given between -180 and +180.  (If given in degrees.  More traditionally, it would be given in hours.)

Reply:
This is the code lifted from `sims_GalSimInterface` (we can refactor later).  I originally had that code make sure `ha` was in the range (0, 360), but people complained that that wasn't necessary.  I don't think it's worth revisiting this issue given the previous discussion on this exact point.